packages/bindings-test/src/multitest.rs

```diff
@@ -153,6 +154,42 @@ impl Pool {
         }
     }
 
+    // returns spot price as place holders, not the arithmetic twap value
+    pub fn arithmetic_twap(
+        &self,
+        quote_asset_denom: &str,
+        base_asset_denom: &str,
+    ) -> Result<Decimal, OsmosisError> {
+        let (bal_in, bal_out) = match (
+            self.get_amount(quote_asset_denom),
+            self.get_amount(base_asset_denom),
+        ) {
+            (Some(a), Some(b)) => (a, b),
+            _ => return Err(OsmosisError::AssetNotInPool),
+        };
+        let mult = Decimal::one();
+        let price = Decimal::from_ratio(bal_out * mult, bal_in);
+        Ok(price)
+    }
+
+    // returns spot price as place holders, not the arithmetic twap to now value
+    pub fn arithmetic_twap_to_now(
+        &self,
+        quote_asset_denom: &str,
+        base_asset_denom: &str,
+    ) -> Result<Decimal, OsmosisError> {
+        let (bal_in, bal_out) = match (
```

Comment:
nit: can call `arithmetic_twap` or vice versa since the implementation is identical

Reply:
wdym? Are u referring to `arithmetic_twap` in `OsmosisQuery`?